src/devcenter/azext_devcenter/manual/_params.py

```diff
@@ -313,7 +313,7 @@ def load_arguments(self, _):
             help="The name of a Dev " "Box.",
         )
         c.argument("upcoming_action_id", type=str, help="The upcoming action id.")
-        c.argument("delay_until", help="The delayed action time (UTC).")
+        c.argument("delay_time", help="The delayed timespan from the current scheduled time. Format HH:MM'")
```

Comment:
What happens when a user does not input an appropriate formatted HH:MM? For example, no colon, greater than 2 digits for hours? We should add a validator for this argument

Reply:
The help string may be confusing to someone without knowledge of our service, did we confirm with Varsha about the string?